x/marker/keeper/msg_server.go

```diff
@@ -442,6 +443,51 @@ func (k msgServer) Transfer(goCtx context.Context, msg *types.MsgTransferRequest
 	return &types.MsgTransferResponse{}, nil
 }
 
+// Transfer handles a message to send coins from one account to another (used with restricted coins that are not
+//
+//	sent using the normal bank send process)
+func (k msgServer) IbcTransfer(goCtx context.Context, msg *types.MsgIbcTransferRequest) (*types.MsgIbcTransferResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// Validate transaction message.
+	if err := msg.ValidateBasic(); err != nil {
+		return nil, sdkerrors.ErrInvalidRequest.Wrap(err.Error())
+	}
+
+	from, err := sdk.AccAddressFromBech32(msg.Transfer.Sender)
+	if err != nil {
+		return nil, err
+	}
+	admin, err := sdk.AccAddressFromBech32(msg.Administrator)
+	if err != nil {
+		return nil, err
+	}
+
+	err = k.IbcTransferCoin(ctx, msg.Transfer.SourcePort, msg.Transfer.SourceChannel, msg.Transfer.Token, from, admin, msg.Transfer.Receiver, msg.Transfer.TimeoutHeight, msg.Transfer.TimeoutTimestamp, func(ctx sdk.Context, k ibckeeper.Keeper, sender sdk.AccAddress, token sdk.Coin) (canTransfer bool, err error) {
+		if !k.GetSendEnabled(ctx) {
```

Comment:
This check is not appropriate for a restricted marker. The transfer operation checks the transfer grant has been assigned to a user and thus bypasses this check

Reply:
> This check is not appropriate for a restricted marker. The transfer operation checks the transfer grant has been assigned to a user and thus bypasses this check

that is the ibc module send enabled, not the bank send enabled flag, imo ibc module send enabled should still be honored since it maybe useful to turn off all transfers.